README.md

```diff
@@ -6,5 +6,68 @@
 Virtual Machine agent for hardware virtualized containers
 
 ## Role
-This project holds the code related to a virtual machine agent relying on the communication protocol defined by hyperstart. That way, it allows to spawn some processes on behalf of pod/container(s) running inside the virtual machine.
-The code relies heavily on [libcontainer](https://github.com/opencontainers/runc/tree/master/libcontainer) so that we can reuse as much as possible the code used by `runc` (standard to run containers on bare metal).
+[`agent`](https://github.com/clearcontainers/agent) is a daemon that runs in the
```

Comment:
This section is based on an old version of the new arch doc. Please update it to use the latest:

https://github.com/clearcontainers/runtime/pull/543/files#diff-597ae097ea464788b7b11bb45e54409eR110

In fact, it might make sense to sync with @egernst and @grahamwhaley since if this lands we will end up with two copies of the same text. Do we really want to do this? If so, how are we going to guarantee they stay in sync?

Maybe the arch doc could Agent section could simply reference this text once the PR lands to avoid the duplication?

Reply:
@grahamwhaley What do you think? I think we are anyways duplicating docs in our other repos to some extent.